openfisca_us/parameters/usda/snap/homeless_shelter_deduction.yaml

```diff
@@ -0,0 +1,22 @@
+description: SNAP Homeless Shelter Deduction Parameters
+
+deduction:
+  2021-01-01: 143
+  metadata:
```

Comment:
move with other metadata and add `period: week`

Reply:
week? I don't follow, is there something I'm missing in the law?

Reply:
Yeah it's monthly right? In that case, `period: month`